src/views/Pools/components/CakeVaultCard/UnstakingFeeCountdownRow.tsx

```diff
@@ -22,8 +23,11 @@ const UnstakingFeeCountdownRow: React.FC<UnstakingFeeCountdownRowProps> = ({
     parseInt(lastDepositedTime, 10),
     parseInt(withdrawalFeePeriod, 10),
   )
+  // TODO: @HUTCH please rename the "account" prop above to something else. We use "account" everywhere in
+  // the app to refer to the connected wallet address
+  const { account: walletAddress } = useWeb3React()
```

Comment:
hey, the account passed via prop IS the address. Idk why it defaults to `true`, but its defined as `string` and passed from same `useWeb3React()` to the `<CakeVaultCard />` in the `/Pools/index.tsx`

Reply:
As Cheems said, the default value on the account (true) is wrong, the account is a string and it's passed from the useWeb3React , so this line is useles.